src/routing_node.rs

```diff
@@ -677,18 +677,24 @@ impl<F> RoutingNode<F> where F: Interface {
         Ok(())
     }
 
+    /// Every node needs to republish its public key directly to PublicPmid.name every 10 minutes.
+    /// Sentinel will query this pool.  No handle_get_public_pmid is needed.
```

Comment:
Can you remove above comment. there is no api for PutPublicPmid...
This only happens while bootstrapping.